Styling/README.md

```diff
@@ -640,6 +644,54 @@ Returns the square root of value if value >= 0. Returns NaN when value < 0.
 }
 ```
 
+#### atan2
+
+`atan2(value1 : Number, value2 : Number) : Number`
+
+Returns the angle `theta` in polar coordinates `(r, theta)` for a point with Cartesian coordinates `(value1, value2)`.
+
+```json
+{
+    "show" : "atan2(${Angle}, ${temperature}) > 0.0"
+}
+```
+
+#### pow
+
+`pow(value1 : Number, value2 : Number) : Number`
+
+Returns `value1` raised to the power of `value2`.
+
+```json
+{
+    "show" : "pow(${Angle}, ${temperature}) > 1.0"
```

Comment:
Instead of `Angle`, maybe something like `Density`. For consistency upper-case `temperature`.